contracts/src/v0.8/l2ep/dev/scroll/ScrollSequencerUptimeFeed.sol

```diff
@@ -41,6 +41,8 @@ contract ScrollSequencerUptimeFeed is
   error InvalidSender();
   /// @notice Replacement for AggregatorV3Interface "No data present"
   error NoDataPresent();
+  /// @notice Address must not be the zero address
+  error ZeroAddress(string msg);
```

Comment:
Let's remove the string arg here, that takes up a lot of gas and contract size. The error name has enough info